packages/block-library/src/navigation/use-navigation-menu.js

```diff
@@ -67,22 +54,26 @@ export default function useNavigationMenu( ref ) {
 		},
 		[ ref, permissions ]
```

Comment:
Let's try to remove `permissions` from this `useSelect`. None of them need to be included and because `useSelect` establishes a subscription to the store the entire select will be re-run each time a permission changes even if that has nothing to do with the other queries.

We can safely decouple the permissions bit from the main `useSelect`.